components/automate-chef-io/content/docs/teams.md

```diff
@@ -11,172 +11,29 @@ toc = true
     weight = 60
 +++
 
-{{< info >}}
-This content is specific to IAM v1. See [IAM v2 Overview]({{< ref "iam-v2-overview.md#" >}}) for IAM v2 specific information.
-{{< /info >}}
+## Overview
+This guide will show you how to manage Chef Automate teams. You can import existing teams into Chef Automate with [Microsoft AD (LDAP)]({{< ref "configuration.md#microsoft-active-directory" >}}), [generic LDAP]({{< ref "configuration.md#ldap" >}}), or [SAML]({{< ref "configuration.md#saml" >}}). You can also create local Chef Automate teams that are independent of LDAP or SAML.
 
-This guide will show you how to manage Chef Automate teams. Import existing teams into Chef Automate with [Microsoft AD (LDAP)]({{< ref "configuration.md#microsoft-active-directory" >}}), [generic LDAP]({{< ref "configuration.md#ldap" >}}) or [SAML]({{< ref "configuration.md#saml" >}}).
+Permission on the `iam:teams` resource is required to interact with teams. Any user that is part of the `admins` team or the `Administrator` policy will have this permission. Otherwise, [IAM v2 custom policies]({{< relref "iam-v2-guide.md#creating-custom-policies" >}}) can be created to assign this permission.
```

Comment:
I do not think we should really be talking about a _resource_ here since resources are _personae non gratae_. Especially since the end of the paragraph references "this permission". But what permission? Something like this would help users understand a bit better: "...actions under `iam:teams`, which could include `iam:teams:get`, `iam:teams:list`, etc., or just `iam:teams:*` for full access".

Same concern about users below.